k2/csrc/ragged_ops_inl.h

```diff
@@ -667,6 +668,28 @@ Ragged<T> CreateRagged2(const std::vector<std::vector<T>> &vecs) {
   return Ragged<T>(shape, values_array);
 }
 
+template <typename T>
+Array2<T> PadRagged(Ragged<T> &src, T padding_value) {
+  NVTX_RANGE(K2_FUNC);
+  K2_CHECK_EQ(src.NumAxes(), 2);
+  ContextPtr c = src.Context();
+  int32_t row_num = src.Dim0(),
+          col_num = src.shape.MaxSize(1);
+  Array2<T> res(c, row_num, col_num, padding_value);
```

Comment:
We don't need to set a deault value here as it involves an extra kernel .

We can use K2_EVAL2 in the following.